compute_sdk/globus_compute_sdk/sdk/executor.py

```diff
@@ -148,16 +149,24 @@ def __init__(
         for key in kwargs:
             if key in deprecated_kwargs:
                 warnings.warn(
-                    f"`{key}` is not utilized and will be removed in a future release",
+                    f"'{key}' is not utilized and will be removed in a future release",
                     DeprecationWarning,
                 )
                 continue
             msg = f"'{key}' is an invalid argument for {self.__class__.__name__}"
             raise TypeError(msg)
 
-        if not funcx_client:
-            funcx_client = Client()
-        self.funcx_client = funcx_client
+        if funcx_client:
+            warnings.warn(
+                "'funcx_client' is superceded by 'client'"
+                " and will be removed in a future release",
+                DeprecationWarning,
+            )
+
+        if not client:
+            client = funcx_client if funcx_client else Client()
+
+        self.client = client
```

Comment:
this should still get the value of funcx_client if it's used, right?

Reply:
oh it does, was just getting confused about the handling when both are set. back to bed for me.